operator/option/config.go

```diff
@@ -385,7 +386,8 @@ type OperatorConfig struct {
 	// PodRestartSelector specify the labels contained in the pod that needs to be restarted before the node can be de-stained
 	PodRestartSelector string
 
-	// AWSPaginationEnabled toggles pagination for AWS EC2 API requests
+	// Deprecated: AWSPaginationEnabled is deprecated and a no-op in v1.19.
+	// It will be removed in v1.20. Pagination now happens automatically.
 	AWSPaginationEnabled bool
```

Comment:
I think this (and the corresponding assignment in `(*OperatorConfig).Populate` can already be removed since it is only set but never read after this PR.